bin/run-tests

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+base="bundle exec rspec -f progress -f JUnit -o spec/ruby-agent-junit.xml"
```

Comment:
@kuisathaverat We probably want to separate the junit outputs too but I'm unsure what's possible in Jenkins.